### PR TITLE
Updates _code.scss to follow BEM methodology

### LINKED ISSUE #829: Refactor _code.scss with BEM
# Problem
Refactor _code.scss with BEM.

## References

Progress on: #672

### PR DESCRIPTION
**Pull request by marineb**
# What's in this PR?

- Updates _code.scss to follow BEM methodology
- Fixes one liners to also get the theme colors
- Didn't touch the highlight styles

## References
Fixes #829 

Progress on: #672

### CODE REVIEW

File: app/styles/_code.scss [hunk 1]
```diff
@@ -28,7 +19,14 @@ code {
   padding: 4px 4px 2px 4px;
 }
 
-.light {
+.code--theme-dark {
+  pre, code {
```
Comment (WenInCode) [inline app/styles/_code.scss]: I think you can nest this under the other `.code--theme-dark` rule below, rather than having 2

**Comment by marineb** [comment]
Thank you! I've made the 2 updates.

**Comment by WenInCode** [comment]
@marineb the changes you made look good 

For the `code-theme-selector` you'll need to update this style sheet with the new style names for it to show on the comment form.
- https://github.com/marineb/code-corps-ember/blob/develop/app/styles/components/code-theme-selector.scss#L6
- https://github.com/marineb/code-corps-ember/blob/develop/app/styles/components/code-theme-selector.scss#L10

After those changes I would say this is good to go!

**Comment by marineb** [comment]
The dark/light toggle does show up for me.
I left it this way in the code for now, on purpose, since I was only tackling _code.scss in this PR. (in my first commit, I had changed it, but then realized it was beyond the scope). 

Unless I'm wrong? Sorry in advance, I'm a total beginner.

**Comment by marineb** [comment]
Whoops. I setup docker, and now, I'm able to test it correctly. I see what's happening. Will send back! Sorry about that!!

**Comment by marineb** [comment]
Should be good now.

**Comment by WenInCode** [comment]
@marineb can you rebase of the latest `develop` and then we'll merge this! (probably in the morning because I am going to sleep 😛 )

**Comment by marineb** [comment]
Awesome. I just rebased against `upstream/develop`.

**Comment by JoshSmith** [comment]
🙌

**Pull request merged**